src/main/java/edu/ucdavis/dss/ipa/api/components/course/CourseViewController.java

```diff
@@ -146,6 +146,24 @@ public void deleteCourse(@PathVariable long courseId, HttpServletResponse httpRe
 		courseService.delete(courseId);
 	}
 
+	@RequestMapping(value = "/api/courseView/schedules/{workgroupId}/{year}/courses", method = RequestMethod.PUT, produces="application/json")
+	@ResponseBody
+	public List<Long> deleteMultipleCourses(@PathVariable long workgroupId, @PathVariable long year, @RequestBody List<Long> courseIds, HttpServletResponse httpResponse) {
+		Schedule schedule = scheduleService.findByWorkgroupIdAndYear(workgroupId, year);
+
+		if (schedule == null) {
+			httpResponse.setStatus(HttpStatus.BAD_REQUEST.value());
```

Comment:
404 is more appropriate. Bad request would be if the request isn't understood, while 404 means the request is understood (e.g. a workgroupId and year were given) but that there's no data to be found.

Reply:
Hmmmm, you are correct. 404 feels odd, but it is definitely more appropriate to this situation of 'target resource not found'